tips/migrations/0002_alter_tip_author_alter_tip_text.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.0.3 on 2022-04-11 19:18
+
```

Comment:
Can you recreate the migration files so we will have only the init file?
To do it you need to ease the current tips migration file and the db
then run: python manage.py makemigrations and then migrate

Reply:
Got it